.github/workflows/release.yml

```diff
@@ -81,6 +85,53 @@ jobs:
           if [ $PROJECT_NAME == "taichi-nightly" ]; then export PYPI_PWD="$NIGHT_PWD" && python build.py upload --skip_build --testpypi --project_name $PROJECT_NAME
           elif [ $PROJECT_NAME == "taichi" ]; then export PYPI_PWD="$PROD_PWD" && python build.py upload --skip_build; fi
 
+  build_and_upload_m1:
+    name: Build and Upload (Apple M1)
+    needs: matrix_prep
+    strategy:
+      matrix: ${{ fromJson(needs.matrix_prep.outputs.matrix_osx) }}
+    runs-on: [self-hosted, m1]
+    defaults:
+      run:
+        shell: /bin/bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Build
+        run: |
+          conda activate $PYTHON
```

Comment:
if conda activate worked, the python executable should have been callable without adding to PATH?